### PR TITLE
adding missing integration option

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

- added option to disable integration and still have the forms working.

### CODE REVIEW

File: src/Integrations/Mailer/Mailer.php [hunk 1]
```diff
@@ -81,48 +81,34 @@ public function fallbackEmail(array $data): bool
...
+			unset($data['files']);
 		}
+		if (isset($data['subject'])) {
+			unset($data['subject']);
+		}
+		if (isset($data['isDisabled'])) {
+			unset($data['isDisabled']);
+		}
+
+		$body .= '<pre style="white-space: pre-wrap; word-wrap: break-word; font-family: monospace;">' . \htmlentities(\wp_json_encode($data, \JSON_PRETTY_PRINT | \JSON_UNESCAPED_SLASHES), \ENT_QUOTES, 'UTF-8') . '</pre>';
```
Comment (dadadavorin) [inline src/Integrations/Mailer/Mailer.php:111]: putting the ChatGPT to use 😄

File: src/Integrations/Moments/SettingsMoments.php [hunk 2]
```diff
@@ -148,59 +150,84 @@ public function getSettingsGlobalData(): array
...
-								'inputValue' => !empty($apiPassword) ? 'xxxxxxxxxxxxxxxx' : $this->getOptionValue(self::SETTINGS_MOMENTS_API_PASSWORD_KEY),
-								'inputIsDisabled' => !empty($apiPassword),
+								'component' => 'checkboxes',
+								'checkboxesFieldLabel' => '',
+								'checkboxesName' => $this->getSettingsName(self::SETTINGS_MOMENTS_SKIP_INTEGRATION_KEY),
+								'checkboxesContent' => [
+									[
+										'component' => 'checkbox',
+										'checkboxLabel' => \__('Deactivate integration and send all the data to the fallback email.', 'eightshift-forms'),
+										'checkboxHelp' => \__('If you choose to activate this option, the form integration will be disabled and all data will be sent to the fallback email address set for the form.', 'eightshift-forms'),
```
Comment (dadadavorin) [inline src/Integrations/Moments/SettingsMoments.php:160]:
```suggestion
										'checkboxHelp' => \__('If you choose to activate this option, the form integration will be disabled and all the data will be sent to the fallback email address set for the form.', 'eightshift-forms'),
```

File: src/Integrations/Moments/SettingsMoments.php [hunk 3]
```diff
@@ -148,59 +150,84 @@ public function getSettingsGlobalData(): array
...
+									'inputIsDisabled' => !empty($apiPassword),
+								],
+								[
+									'component' => 'divider',
+									'dividerExtraVSpacing' => true,
+								],
+								[
+									'component' => 'submit',
+									'submitFieldSkip' => true,
+									'submitValue' => \__('Test api connection', 'eightshift-forms'),
```
Comment (dadadavorin) [inline src/Integrations/Moments/SettingsMoments.php:223]:
```suggestion
									'submitValue' => \__('Test API connection', 'eightshift-forms'),
```

**Review by dadadavorin — APPROVED** [review]
👍 (just a couple of grammar suggestions)

File: src/Integrations/Moments/SettingsMoments.php (hunk 3 above)
Comment (iruzevic) [reply src/Integrations/Moments/SettingsMoments.php:223]: this button is on multiple locations so I will rename it on all in the next PR tnx for the comment

File: src/Integrations/Mailer/Mailer.php (hunk 1 above)
Comment (iruzevic) [reply src/Integrations/Mailer/Mailer.php:111]: oooo yes :D

**Pull request merged**